openshift/4.0/templates/api/deploy.yaml

```diff
@@ -139,37 +139,25 @@ parameters:
     description: "Maximum amount of memory the container can use."
     value: "4Gi"
 objects:
-  # Allow the api to make requests to the internet.
```

Comment:
nps only act on ingress, and so no np is required to allow pod-to-internet communication.